README.md

```diff
@@ -55,7 +55,50 @@ For most existing Dojo 1 Dijits, the TypeScript typings can be found at [dojo/ty
 
 ### ReduxInjector
 
-*Coming Soon*
+`ReduxInjector` can be used to bind a redux store and Dojo 2 widgets using the `registry`.
+
+An injector can be defined in the registry, which is then registered with the `Router`, and provided to the `Projector` as one of its properties. This is demonstrated in the example below.
```

Comment:
I don't understand this sentence, the `Router` doesn't have anything to do with the using `ReduxInjector`...